### DISCUSSION TITLE
Can i get clean build?

### DISCUSSION THREAD

**Discussion by Alone0316** [ROM-builders/temporary#14118] [General]
Can i get a clean build of Lineage? the trees seem fine i am able to get successful build with the same tree, link to successful build https://cirrus-ci.com/build/6281322862739456
@Apon77

**Comment by Apon77** [comment]
No. It says no information about lineage need clean build

**Reply by Alone0316** [reply]
Lineage need clean build because i m facing a weird problem with Lineage even though trees are same for both and clean build would solve it
